docs/en/observability/tab-widgets/deploy-ess-stack.asciidoc

```diff
@@ -0,0 +1,23 @@
+// tag::ess[]
+. {ess-trial}[Get a free trial].
+
+. Log into {ess-console}[Elastic Cloud].
+
+. Click *Create deployment*.
+
+. Select *Elastic Observability* and give your deployment a name.
+
+. Click *Create deployment* and copy the password for the `elastic` user.
```

Comment:
We should use the [shared file](https://github.com/elastic/docs/blob/master/shared/cloud/ess-getting-started-obs.asciidoc) so that we only have to update one thing.
```suggestion
include::{docs-root}/shared/cloud/ess-getting-started-obs.asciidoc[]
```

Reply:
Ah, good spot! :-)